src/elfpy/utils/time.py

```diff
@@ -0,0 +1,34 @@
+"""
+Helper functions for converting time units
+"""
+
+
+import datetime
+import pytz
+
+import numpy as np
+
```

Comment:
It would be good to add time stretching functions here.

Reply:
Done.